instrumentation/httpclient5/src/main/java/brave/httpclient5/HttpClientUtils.java

```diff
@@ -58,8 +58,8 @@ static boolean isLocalCached(HttpContext context, Span span) {
     if (!HAS_CLIENT_CACHE_SUPPORT) {
       return false;
     }
-    boolean cacheHit = CacheResponseStatus.CACHE_HIT == context.getAttribute(
-      HttpCacheContext.CACHE_RESPONSE_STATUS);
+    final HttpCacheContext cacheContext = HttpCacheContext.cast(context);
```

Comment:
One q is.. is this type available in older versions than what we test? If not update the readme to say minimum version

Reply:
Thanks @codefromthecrypt , the docs says `HttpCacheContext`  is` * @since 4.3` , we should be good here :)